shared/common-adapters/copy-text.tsx

```diff
@@ -112,17 +108,17 @@ const styles = Styles.styleSheetCreate(
           height: undefined,
           marginLeft: 'auto',
           minWidth: undefined,
-          paddingLeft: 17,
```

Comment:
xsmall is 12, which is significantly smaller than 17. Intentional?

Reply:
Yeah all these are intentional; Cecile wanted to shrink this component a bit globally